cli/src/test/java/com/devonfw/tools/ide/tool/intellij/IntellijTest.java

```diff
@@ -126,6 +126,7 @@ public void testIntellijRun(String os, WireMockRuntimeInfo wmRuntimeInfo) throws
     SystemInfo systemInfo = SystemInfoMock.of(os);
     this.context.setSystemInfo(systemInfo);
     Intellij commandlet = new Intellij(this.context);
+    this.context.info("Starting testIntellijRun on {}", os);
```

Comment:
I was adding this because I was desperate reading the logs on github PR build and not getting what test and OS actually caused the error.
Theoretically I could also be removed now since I was able to trace down the bug this way.
However, I am more with you to think about adding this to all these context-based tests.

Maybe we should have a look together at all of these tests and centralize a lot of such things (also asserting the same things for multiple tools, etc.). On the other hand I think that sometimes tests can get more complex to read and understand if we have too much generic stuff and over-engineer them.